deepstack_sdk/utils.py

```diff
@@ -25,4 +25,8 @@ def bytesToPIL(image_data):
     
 def bytestoCV2(image_data):
     np_arr = numpy.asarray(bytearray(image_data),dtype=numpy.uint8)
-    return cv2.imdecode(np_arr,cv2.IMREAD_COLOR)
\ No newline at end of file
+    return cv2.imdecode(np_arr,cv2.IMREAD_COLOR)
+
+def frameTracker(log, frame_count):
+    if log:
+        print("Frame {} processed".format(frame_count))
```

Comment:
The message here should be something like `Processing frame {}` considering the function is called before actual detection is performed.